server/service/osquery_utils/queries.go

```diff
@@ -363,6 +363,10 @@ FROM logical_drives WHERE file_system = 'NTFS' LIMIT 1;`,
 		Query:            `select version from munki_info;`,
 		DirectIngestFunc: directIngestMunkiInfo,
 	},
+	"google_chrome_profiles": {
+		Query:            `SELECT email FROM google_chrome_profiles`, // TODO(mna): where not ephemeral?
```

Comment:
Hm, good question. Reading https://support.google.com/chrome/a/answer/3538894?hl=en makes me thing that yes, we want to skip ephemeral profiles, because they point is to help map a device to an employee.